cg/models/taxprofiler/taxprofiler.py

```diff
@@ -30,7 +30,7 @@ class TaxprofilerParameters(WorkflowParameters):
     perform_runmerging: bool = True
     run_kraken2: bool = True
     kraken2_save_reads: bool = True
-    kraken2_save_readclassification: bool = True
+    kraken2_save_readclassifications: bool = True
```

Comment:
```suggestion
    kraken2_save_read_classifications: bool = True
```

Reply:
perhaps out of scope for this pr, if it changes things in too many places, ignore this.

Reply:
That is the argument used when running the taxprofiler pipeline https://github.com/nf-core/taxprofiler/blob/master/nextflow.config#L122
Resolving this.